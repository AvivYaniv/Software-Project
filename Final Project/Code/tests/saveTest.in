solve tests/b2_2.txt
autofill
autofill
validate
save tests/observed/save_solve_board.txt
edit tests/b2_2.txt
set 1 1 0
set 2 1 0
set 3 1 0
set 4 1 0
set 2 2 3
set 3 3 1
save tests/observed/save_edit_board.txt
solve tests/b2_2.txt
set 2 2 2
save tests/observed/save_board2.txt
undo
set 3 2 3
validate
save tests/observed/save_board3.txt
edit tests/b2_2.txt
set 3 2 3
validate
save tests/observed/save_board4.txt
set 2 2 2
save tests/observed/save_board5.txt
solve tests/b2_2.txt
save tests/path/that/does/not/exist.txt
set 2 4 2
save tests/observed/save_test_with_error.txt
solve tests/observed/save_test_with_error.txt
print_board
mark_errors 0
solve tests/observed/save_test_with_error.txt
print_board
edit tests/observed/save_test_with_error.txt
print_board
save tests/observed/save_test_with_error2.txt
undo
exit