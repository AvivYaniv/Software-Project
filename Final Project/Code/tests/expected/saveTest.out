Sudoku
------
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |      1.|
-------------------
|        |      2.|
|        |        |
-------------------
Enter your command:
Cell <4,1> set to 3
-------------------
|  1.  2.|  4.  3 |
|        |      1.|
-------------------
|        |      2.|
|        |        |
-------------------
Enter your command:
Cell <3,2> set to 2
Cell <4,4> set to 4
-------------------
|  1.  2.|  4.  3 |
|        |  2   1.|
-------------------
|        |      2.|
|        |      4 |
-------------------
Enter your command:
Validation passed: board is solvable
Enter your command:
Saved to: tests/observed/save_solve_board.txt
Enter your command:
-------------------
|  1   2 |  4     |
|        |      1 |
-------------------
|        |      2 |
|        |        |
-------------------
Enter your command:
-------------------
|      2 |  4     |
|        |      1 |
-------------------
|        |      2 |
|        |        |
-------------------
Enter your command:
-------------------
|        |  4     |
|        |      1 |
-------------------
|        |      2 |
|        |        |
-------------------
Enter your command:
-------------------
|        |        |
|        |      1 |
-------------------
|        |      2 |
|        |        |
-------------------
Enter your command:
-------------------
|        |        |
|        |      1 |
-------------------
|        |      2 |
|        |        |
-------------------
Enter your command:
-------------------
|        |        |
|      3 |      1 |
-------------------
|        |      2 |
|        |        |
-------------------
Enter your command:
-------------------
|        |        |
|      3 |      1 |
-------------------
|        |  1   2 |
|        |        |
-------------------
Enter your command:
Saved to: tests/observed/save_edit_board.txt
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |      1.|
-------------------
|        |      2.|
|        |        |
-------------------
Enter your command:
-------------------
|  1.  2.|  4.    |
|      2*|      1.|
-------------------
|        |      2.|
|        |        |
-------------------
Enter your command:
Saved to: tests/observed/save_board2.txt
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |      1.|
-------------------
|        |      2.|
|        |        |
-------------------
Undo 2,2: from 2 to _
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |  3   1.|
-------------------
|        |      2.|
|        |        |
-------------------
Enter your command:
Validation failed: board is unsolvable
Enter your command:
Saved to: tests/observed/save_board3.txt
Enter your command:
-------------------
|  1   2 |  4     |
|        |      1 |
-------------------
|        |      2 |
|        |        |
-------------------
Enter your command:
-------------------
|  1   2 |  4     |
|        |  3   1 |
-------------------
|        |      2 |
|        |        |
-------------------
Enter your command:
Validation failed: board is unsolvable
Enter your command:
Error: board validation failed
Enter your command:
-------------------
|  1   2*|  4     |
|      2*|  3   1 |
-------------------
|        |      2 |
|        |        |
-------------------
Enter your command:
Error: board contains erroneous values
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |      1.|
-------------------
|        |      2.|
|        |        |
-------------------
Enter your command:
Error: File cannot be created or modified
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |      1.|
-------------------
|        |      2.|
|      2*|        |
-------------------
Enter your command:
Saved to: tests/observed/save_test_with_error.txt
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |      1.|
-------------------
|        |      2.|
|      2*|        |
-------------------
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |      1.|
-------------------
|        |      2.|
|      2*|        |
-------------------
Enter your command:
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |      1.|
-------------------
|        |      2.|
|      2 |        |
-------------------
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |      1.|
-------------------
|        |      2.|
|      2 |        |
-------------------
Enter your command:
-------------------
|  1   2*|  4     |
|        |      1 |
-------------------
|        |      2 |
|      2*|        |
-------------------
Enter your command:
-------------------
|  1   2*|  4     |
|        |      1 |
-------------------
|        |      2 |
|      2*|        |
-------------------
Enter your command:
Error: board contains erroneous values
Enter your command:
Error: no moves to undo
Enter your command:
Exiting...
